addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onkyo"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- OnkyoAVR Thing Type -->
+    <thing-type id="onkyoAVR">
+        <label>Onkyo AV Receiver</label>
+        <description>Network enabled Onkyo AV Receivers</description>
+
+        <channels>
+            <channel id="power" typeId="power"/>
+            <channel id="input" typeId="input"/>
+            <channel id="volume" typeId="volume"/>
+            <channel id="mute" typeId="mute"/>
+            <channel id="next" typeId="next" />
+            <channel id="prev" typeId="prev" />
+            <channel id="control" typeId="control" />
+            <channel id="currentPlayingTime" typeId="currentPlayingTime" />
+            <channel id="title" typeId="title" />
+            <channel id="album" typeId="album" />
+            <channel id="artist" typeId="artist" />
+        </channels>
+        
+        <config-description>
+            <parameter name="ipAddress" type="text" required="true">
+                <label>IP or Host Name</label>
+                <description>The IP or host name of the Onkyo Receiver
+                </description>
+            </parameter>
+            <parameter name="port" type="integer" required="true"
+                min="1" max="65335">
+                <label>Onkyo Receiver Port</label>
+                <description>Port of the Onkyo to control</description>
+                <default>60128</default>
+            </parameter>            
+        </config-description>
+    </thing-type>
+
+    <!-- Onkyo unknown Thing Type -->
+    <thing-type id="onkyoUnsupported">
```

Comment:
you can add a `listed="false"` here, which will suppress this entry when manually adding a thing.